command_line_assistant/client.py

```diff
@@ -57,6 +57,13 @@ def main() -> int:
     try:
         stdin = read_stdin()
         args = add_default_command(stdin, sys.argv)
+
+        # In case that the user only calls `chat`, `history` or anything else,
+        # we just print help and return with os.EX_USAGE.
+        if len(args) == 1 and not stdin:
```

Comment:
```suggestion
        if len(args) <= 1 and not stdin and "feedback" not in args:
```

what do you think, @subpop?

Reply:
Yea. This will work. It's not the most scalable solution if we need to add more single command arguments, but I don't think we'll add too many of those.